sysroot/usr/include/kernel/boot/Handoff.hpp

```diff
@@ -0,0 +1,129 @@
+/**
+ * @file Handoff.hpp
+ * @author Keeton Feave (keetonfeavel@cedarville.edu)
+ * @brief 
+ * @version 0.1
+ * @date 2021-06-02
+ * 
+ * @copyright Copyright the Panix Contributors (c) 2021
+ * 
+ */
+#include <stdint.h>
+#include <stdbool.h>
+
+namespace Boot {
+
+enum HandoffFirmwareType {
+    BIOS = 0,
+    UEFI = 1,
+};
+
+enum HandoffBootloaderType {
+    Multiboot2 = 0,
+    Stivale2 = 1,
+};
+
+// Unused for now.
+class HandoffRSDPDescriptor {
```

Comment:
Does this info need to be handed off to the kernel, or can we just preserve the RSDT address?

Reply:
Do we not want to parse the RSDP information? We would still keep the pointer, but I figured having a class around it to access information easier would be nice. Feel free to disagree.

Reply:
Hmm well I was just thinking it would be simpler to not have it, but I guess keep it until we know for sure what we want to do with that information.